### ISSUE TITLE
Profile proposals?

### ISSUE THREAD

**Issue by villares** [CreativeCodeBerlin/creative-coding-minilist#1]
Thanks for the lovely resource!

I was thinking about "I'am a teacher"/"I'm a math teacher", and "I'am a Pythonista". Would that be too much?
Could we discuss profiles on issues and eventually PR them?

A tool for math teachers might be Geogebra, the book could be Peter Farell's  https://nostarch.com/mathadventures, but he uses Processing.Py not Geogebra:(

For the Pythonista profile, the tool could be github.com/berinhard/pyp5js (or some other from these: https://github.com/villares/Resources-for-teaching-programming/) and the artist could be Bernardo Fontes (https://berinhard.github.io/sketches/) or Allison Parrish (https://www.decontextualize.com/)

**Comment by micuat** [comment]
teacher is a good one!

Python may be a bit too specific for a beginner. But Allison is definitely a great pointer. What about "I'm a poet" profile so we can add her as an artist? Feel free to edit the original document if you have concrete ideas (you might need to make an account on hackmd)
https://hackmd.io/QMZVoFwtRqSHuAtuu4qnAQ

**Comment by villares** [comment]
So, I had a hackmd account, and I have just tried to add a **poet profile** with Allison as the chosen artist, check if you can see it...
I think a Teacher profile would be great but I have to think harder... :D

**Comment by SableRaf** [comment]
I saw the poet profile! I was wondering who added it :D Nice. I didn't know about Thonny either. What makes it good for poets?

**Comment by villares** [comment]
Well, it is debatable if it is any good for poets, but as Allison uses mainly Python, and the book I selected also uses some Python it is a good IDE for beginners that comes with a Python interpreter built-in which avoids some hassle of installing Python 3 (specially on windows). I wouldn't mind if someone has a more poetic tool to select (maybe an easy NLP or text generation library?)

**Comment by micuat** [comment]
Thanks for the suggestions! I’m not sure if we go to python / ml direction or simply using words in a “creative” way - for example, although she’s already in the list, Joana uses words with HTML5 and it’s a beautiful way of working with texts. Movements like this is also an interesting reference
https://taper.badquar.to/5/about.html

**Comment by SableRaf** [comment]
If we could suggest Thonny + some beginner-friendly Python library or tutorial specifically for poetry, that'd be good

**Comment by villares** [comment]
I agree, it would be way better. Let's see if we can find something. 
I was looking into the tools Allison uses, but some are discontinued :( 
http://www.decontextualize.com/teaching/rwet/recursion-and-context-free-grammars/
Let's see what we can find...

**Comment by micuat** [comment]
maybe we can ask @aparrish directly if she has any suggestion for library/tool/book for poets who are interested in creative coding

**Comment by aparrish** [comment]
Hi! Not sure what this is all about but I'd probably direct you to the syllabus and course materials for two of my classes: [Reading and Writing Electronic Text](https://rwet.decontextualize.com/schedule/) and [Computational Approaches to Narrative](http://catn.decontextualize.com/schedule/), both of which are targeted at beginners (though they do require getting into Python and Jupyter Notebooks a bit). The [Electronic Literature Collection](https://collection.eliterature.org/) is a great place to look for computational/digital poetry projects. Hope this helps?
